client/src/components/BottomNav/IconButton.js

```diff
@@ -1,9 +1,9 @@
 import React from "react";
 import PropTypes from "prop-types";
 
-const IconButton = ({ iconSrc, iconAlt }) => {
+const IconButton = ({ iconSrc, iconAlt, onClick }) => {
   return (
-    <button>
+    <button onClick={onClick}>
```

Comment:
Not necessary, but maybe you could name the function `handleOnClick` and the property `onClick` to distinguish the two (I think Leon mentioned this once?!)

Reply:
good point, I think for now I'll leave since it's only the useState and in the function itself I will write a handleClick later on...